src/components/utils/index.tsx

```diff
@@ -0,0 +1,59 @@
+export const number: { num: string }[] = [
```

Comment:
It is a good practice to put array name in plural form to indicate that it's a variable containing many elements

Reply:
ok it is fixed